### PR TITLE
Added the login page

### PR DESCRIPTION
**Pull request by hussiantabha**
image

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
Everything looks clean, good to merge

File: css/login.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.login-container {
+  max-width: 700px;
```
Comment (ozairkhan2810) [inline css/login.css]: can use rem for consistency

File: css/login.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+.login-input {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  padding: 0.5rem 2rem;
+  margin-top: 1.75rem;
+}
+.login-input-text {
+  border: 1px solid #dcdcdc;
```
Comment (ozairkhan2810) [inline css/login.css]: can use variable name for color

**Review by ozairkhan2810 — APPROVED** [review]
UI looks nice
refactor couple of things and then good to merge

**Pull request merged**